README.md

```diff
@@ -120,3 +120,59 @@ We highly recommend setting up your editor to automatically fix linting issues,
 ### Our recommendation
 
 In dotkom, we find that [PyCharm](https://www.jetbrains.com/pycharm/) is a great IDE that is well suited for contributing to onlineweb4. It'll come with support for ESLint, stylelint, and PEP 8 out of the box, and can be set up to run isort and Flake8 with some ease. We recommend it to our beginners who don't want to spend a lot of time setting up the plugins or extensions mentioned above, or don't have any preferences of their own yet. You can apply for a free student license [here](https://www.jetbrains.com/student/).
+
+## API  
+Onlineweb4 comes with an API located at `/api/v1`.  
+Autogenerated Swagger-docs can be found at `/api/v1/docs`.  
+Some endpoints require user authentication. See _Oauth/OIDC_.  
+
+## OAUTH/OIDC  
+Onlineweb4 has an Oauth2/OIDC-provider built-in, allowing external projects to authenticate users through Onlineweb4.  
+[Oath: Which flow should I use?](https://auth0.com/docs/authorization/which-oauth-2-0-flow-should-i-use)  
```

Comment:
```suggestion
[OAuth: Which flow should I use?](https://auth0.com/docs/authorization/which-oauth-2-0-flow-should-i-use)  
```

Reply:
I meant to prefix the title with the authoring organization, Auth0 😅 
Not sure how that became Oath, but I will correct it